bibliopixel/util/project.py

```diff
@@ -0,0 +1,9 @@
+from . importer import make_object
+
+
+class Project(object):
+    def __init__(self, driver, led, animation, run):
```

Comment:
I have one single complaint here.... You should be able to pass any of these options:
- String file path to JSON file to load
- String JSON data
- Python dict object

Ideally we'd then also be able to do something like:
```
sudo python -m bibliopixel.util.project project_definition.json
```